### PR TITLE
Update use-dkim-to-validate-outbound-email.md

### PR DESCRIPTION
**Pull request by puneethmeister**
The above wording of default domain is causing confusion for customers, we need to replace it with MOERA or initial domain i.e. onmicrosoft domain
https://o365exchange.visualstudio.com/IP%20Engineering/_workitems/edit/2074507?src=WorkItemMention&src-action=artifact_link

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email.md [hunk 1]
```diff
@@ -293,9 +293,9 @@ Disabling the signing policy does not completely disable DKIM. After a period of
 ## Default behavior for DKIM and Microsoft 365
 <a name="DefaultDKIMbehavior"> </a>
 
-If you do not enable DKIM, Microsoft 365 automatically creates a 1024-bit DKIM public key for your default domain and the associated private key which we store internally in our datacenter. By default, Microsoft 365 uses a default signing configuration for domains that do not have a policy in place. This means that if you do not set up DKIM yourself, Microsoft 365 will use its default policy and keys it creates to enable DKIM for your domain.
+If you do not enable DKIM, Microsoft 365 automatically creates a 1024-bit DKIM public key for your MOERA/Initial domain and the associated private key which we store internally in our datacenter. By default, Microsoft 365 uses a default signing configuration for domains that do not have a policy in place. This means that if you do not set up DKIM yourself, Microsoft 365 will use its default policy and keys it creates to enable DKIM for your domain.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email.md]:
```suggestion
If you do not enable DKIM, Microsoft 365 automatically creates a 1024-bit DKIM public key for your Microsoft Online Email Routing Address (MOERA)/initial domain and the associated private key which we store internally in our datacenter. By default, Microsoft 365 uses a default signing configuration for domains that do not have a policy in place. This means that if you do not set up DKIM yourself, Microsoft 365 will use its default policy and keys it creates to enable DKIM for your domain.
```

File: microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email.md [hunk 2]
```diff
@@ -293,9 +293,9 @@ Disabling the signing policy does not completely disable DKIM. After a period of
...
 
-Also, if you disable DKIM signing after enabling it, after a period of time, Microsoft 365 will automatically apply the default policy for your domain.
+Also, if you disable DKIM signing on custom domain after enabling it, after a period of time, Microsoft 365 will automatically apply the MOERA/Initial domain policy for your custom domain.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email.md]:
```suggestion
Also, if you disable DKIM signing on your custom domain after enabling it, after a period of time, Microsoft 365 will automatically apply the MOERA/initial domain policy for your custom domain.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@puneethmeister Please review and implement the suggested changes. Thanks

**Comment by puneethmeister** [comment]
@yogkumgit committed the suggested changes

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks

**Comment by puneethmeister** [comment]
@MSFTTracyP Gentle reminder

**Review by MSFTTracyP — COMMENTED** [review]
Thanks

**Review by MSFTTracyP — COMMENTED** [review]
Thanks

**Pull request merged**